assets/index.html

```diff
@@ -0,0 +1,18 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="UTF-8">
+        <title>Node Hello world = example</title>
+    </head>
+    <body>
+        <form action="http://localhost:3000/userform" method="GET">
```

Comment:
remove domain from action ;) leave just `/userform` - when you change domain of your application you will have to change this part also ;)